### DISCUSSION TITLE
How to make changes to the gitbook.io files

### DISCUSSION THREAD

**Discussion by ArslanYM** [AppFlowy-IO/AppFlowy#2509] [General help wanted] [answered]
I cant seem to find the files. I need to make a change in the gitbook documentation guide

**Comment by annieappflowy** [comment, marked as answer]
Hi @ArslanYM , you can submit a PR to request changes to a GitBook page via GitHub
https://github.com/AppFlowy-IO/AppFlowy-Docs

**Reply by ArslanYM** [reply]
@annieappflowy Understood.
